.github/workflows/compile.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         path: executables
     - 
       name: move files one folder up
-      run: mv */*.tar . && find . -type d -empty -delete
+      run: mv */*.tar.gz . && find . -type d -empty -delete
```

Comment:
Why do we need this?

Reply:
just changed the file ending, no idea why the command itself is in here to calculate the checksum

Reply:
ok I see